src/sgame/sg_bot_local.h

```diff
@@ -88,6 +90,37 @@ struct botSkill_t
 	float aimShake;
 };
 
+// boolean flags that tells which skill (compétence) the bot has
+enum bot_skill
```

Comment:
why snake case?

Reply:
Habbits, readability and the consistency with the C++ stdlib I think.

I've tried reverting that but this makes it needlessly painful to deal with conflicts when rebasing.

Reply:
I find `git filter-branch` useful for renaming variables (or an equivalent external tool because filter-branch is badly designed blah blah)

Reply:
I'll take note of that, but I don't think I'll be doing this for this PR because I don't feel like this is worth it.